lib/tasks/repo_export.rake

```diff
@@ -0,0 +1,9 @@
+desc "Run sufia export with InvenioRDM specific converters"
+task :repo_export, [:destination] => :environment do |task, args|
+  export_command = "sufia_export --verbose --models GenericFile=InvenioRdmConverter,Collection=InvenioRdmCollectionConverter"
```

Comment:
Should the option be `GenericFile=InvenioRdmRecordConverter`?
```suggestion
  export_command = "sufia_export --verbose --models GenericFile=InvenioRdmRecordConverter,Collection=InvenioRdmCollectionConverter"
```

Reply:
Ah, it should be. Thanks.